src/Auth/Protocol/OAuth.elm

```diff
@@ -11,6 +11,7 @@ import OAuth
 import OAuth.AuthorizationCode as OAuth
 import Process
 import SHA1
+import Share.Config exposing (isDev)
```

Comment:
Think this needs to be removed?

Reply:
Yep, I think you mentioned this earlier.